components/net/bluetooth_thread.rs

```diff
@@ -634,19 +625,17 @@ impl BluetoothManager {
                 None => None,
             };
         }
-
         let message = match value {
-            Some(v) => BluetoothObjectMsg::BluetoothReadValue { value: v },
-            None => send_error!(sender, VALUE_ERROR),
+            Some(v) => Ok(v),
+            None => return drop(sender.send(Err(String::from(VALUE_ERROR)))),
         };
-
-        sender.send(message).unwrap();
+        let _ = sender.send(message);
```

Comment:
I think this is point now to line 615 https://github.com/szeged/servo/pull/25/files#diff-438a3422f5ba8c39dabd8e4e18928c51R615